.betterer.results

```diff
@@ -471,8 +471,9 @@ exports[`better eslint`] = {
       [0, 0, 0, "Unexpected any. Specify a different type.", "3"],
```

Comment:
Are these betterer changes intended?

Reply:
Not sure, as this is only POC draft there are lot's of things that need to be finished.